public/scripts/extensions/regex/style.css

```diff
@@ -1,3 +1,7 @@
+[hidden] {
```

Comment:
This style is not specific enough.
```suggestion
.regex_settings [hidden] {
```